app/controllers/award_badges_controller.rb

```diff
@@ -0,0 +1,20 @@
+class AwardBadgesController < ApplicationController
+
+  def index
+    @projects = Project.where.not(mentee_id: Badge.where(name: [11]).includes(:users).map{|b| b.users.pluck(:id)}.flatten)
```

Comment:
filtering `mentee id` looks very complicated here. How about `Badge.visionary.joins(:users).uniq.pluck("users.id")` it will also choose mentees that already have assigned visionary badge but for me it looks simpler - what do you think?